roles/kubernetes/node/tasks/install_host.yml

```diff
@@ -6,7 +6,7 @@
   retries: 4
   delay: "{{ retry_stagger | random + 3 }}"
   changed_when: false
-  failed_when: "kubelet_task_compare_result.rc < 0 or kubelet_task_compare_result.rc > 2"
+  failed_when: "kubelet_task_compare_result.rc not in [0,1]"
```

Comment:
Are you sure it shouldn't be `"kubelet_task_compare_result.rc not in [0,2]`? rc == 2 will happen when setting up a new node and `/systembindir/kubelet` doesn't exist yet.